src/backend/joanie/core/serializers/client.py

```diff
@@ -720,6 +722,18 @@ def get_target_enrollments(self, order) -> list[dict]:
             context=self.context,
         ).data
 
+    def validate(self, attrs):
```

Comment:
Why not using `validate_has_consent_to_terms` ?

Reply:
Because I was not aware about that! Looks great, I updated the logic.